src/hooks/use-toast.ts

```diff
@@ -0,0 +1,188 @@
+import * as React from 'react';
```

Comment:
Od Reactu 17 u6 není potrebný importovat celý React. Importuje se automaticky https://legacy.reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html